### PR TITLE
fix(layout): update docs

### LINKED ISSUE #1884: Update Layout Descriptions
As part of the Patternfly Training Documentation, I noticed that the descriptions for each layout are written differently. Lets align this wording.

### PR DESCRIPTION
**Pull request by christiemolloy**
close #1884

### CODE REVIEW

File: src/patternfly/layouts/Bullseye/docs/code.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ## Overview
 
-Use a bullseye layout when there is a single child element, and that child is centered both vertically and horizontally in the parent.
+The Bullseye Layout is designed to both horizontally and vertically center a single child element within its parent.
```
Comment (srambach) [inline src/patternfly/layouts/Bullseye/docs/code.md]: You can remove the split infinitive something like this - "The Bullseye Layout is designed to center a single child element both horizontally and vertically within its parent."

File: src/patternfly/layouts/Gallery/docs/code.md [hunk 2]
```diff
@@ -1,6 +1,6 @@
 ## Overview
 
-A Gallery layout makes all of the children a uniform size, the children are displayed horizontally, and they wrap as needed.
+The Gallery Layout is designed so that all of its children are of uniform size, displayed horizontally and wrap responsively.
```
Comment (srambach) [inline src/patternfly/layouts/Gallery/docs/code.md]: Changing to "...display horizontally, and wrap responsively." will make Oxford comma fans happy and keep the clauses parallel.

**Comment by srambach** [comment]
Nice job making these consistent. Just a couple of overly picky grammar fixes :) and also, should all the layout names be lower case to comply with sentence case capitalization?

**Comment by christiemolloy** [comment]
@stacymcauliffe could you check this for grammar and wording :)

**Comment by srambach** [comment]
LGTM! bitmoji

**Comment by stacymcauliffe** [comment]
This looks really great! Very well-written

**Pull request merged**